### DISCUSSION TITLE
Not Responding after Opening

### DISCUSSION THREAD

**Discussion by bgardels** [WinMerge/winmerge#1506] [Q&A]
I have WinMerge 2.16.22-x64 installed on Win10 and it has been working for quite a while.  However, it suddenly stopped working and does not respond immdediately after opening.  The application appears but then I get the hourglass mouse cursor and Task Manager says not responding.  It works Ok if I run as an Admin; however, running it that way is not ideal for me.  I have tried uninstalling and re-installing, it still does not respond after opening.  I have to use Task Manager to end task.  I do not see any clues in the Windows event log.  Anybody else having this issue?

**Comment by Gitoffthelawn** [comment]
Any chance you had used WinMerge to access network resources that are no longer available?

**Reply by bgardels** [reply]
Yes.  Either that or a local file/folder that was removed.  You know of something?

**Reply by Gitoffthelawn** [reply]
> Yes. Either that or a local file/folder that was removed. You know of something?

Yes. Windows (Win10/Win11 in particular, from what I've heard) is notoriously bad at handling unavailable resources.

My recommendation is to:
1. export all your WinMerge settings
2. edit the settings file to remove any unavailable resources (make a backup first, naturally)
3. reset all WinMerge settings using the WinMerge settings UI
4. import your edited WinMerge settings file into WinMerge

I hope this helps you.
